common/http/platform/beast/http.cpp

```diff
@@ -794,6 +794,22 @@ void Stream::CallErrorHandler(
 	server_.RemoveStream(shared_from_this());
 }
 
+void Stream::CallErrorHandler(
+	const error_code &ec, const IncomingRequestPtr &req, IdentifiedRequestHandler handler) {
+	CallErrorHandler(error::Error(ec.default_error_condition(), ""), req, handler);
+}
+
+void Stream::CallErrorHandler(
+	const error::Error &err, const IncomingRequestPtr &req, IdentifiedRequestHandler handler) {
+	stream_active_.reset();
+	handler(
+		req,
+		err.WithContext(
+			req->address_.host + ": " + MethodToString(req->method_) + " " + request_->GetPath()));
```

Comment:
Q: Are we _always_ guaranteed the `address` and `method` is set unconditionally, when called?

Reply:
No, but even invalid values are valid strings. `MethodToString(req->method_)` will just be `"Invalid"`, and `address_.host` will be empty. So as an error string I think it still does the job.